seed/api/v2_1/views.py

```diff
@@ -311,12 +343,16 @@ def update_with_building_sync(self, request, pk):
             )
 
             property_view = result.pop('property_view')
-
+            previous_state = property_view.state
             # passing in the existing propertyview allows it to process the buildingsync file and attach it to the
             # existing propertyview.
             p_status, new_pv_state, new_pv_view, messages = building_file.process(
                 organization_id, cycle, property_view=property_view
             )
+
+            # merge the relationships from the old property state
+            self._merge_relationships(previous_state, new_pv_state)
```

Comment:
This has no side effects, at the moment.  It looks like it is just all stubbed out for later.  In which case, OK.

Reply:
yes, it has to be hooked up when someone can think about how we merge measures and scenarios.